.eslintrc

```diff
@@ -1,11 +1,16 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2018
+    "ecmaVersion": 6
   },
-  "extends": ["eslint:recommended", "eslint-config-prettier"],
+  "extends": [
+    "eslint:recommended",
+    "eslint-config-prettier",
+    "plugin:@typescript-eslint/recommended"
+  ],
   "rules": {
     "no-unused-vars": "off",
-    "no-console": "off"
+    "no-console": "off",
```

Comment:
👀 Leaving a bunch of console logs in your code is not a good idea unless you really have to. I suggest not adding this.

Reply:
Ah yes you're right, I removed it.